prebid-mobile/prebid-mobile.md

```diff
@@ -11,31 +11,93 @@ nav_section: prebid-mobile
-- Increased yield due to increased competition between demand sources
-- A lightweight library that uses simple query strings to send ad requests and to pass results
+## Benefits and Features
 
-### How it Works
+Some of the benefits to using the Prebid Mobile header bidding solution include:
 
-The Prebid Mobile SDK sends requests to Prebid Server to help app developers to access additional demand that is not available from their mobile ad network.
+-   Provides a single integration point with Prebid Server, providing direct access to more mobile buyers.
+-   Server-side configuration management; no need for developers to update the app to make configuration changes.
+-   Prebid Mobile is a transparent, open source solution.
+-   Created for integration with other prebid products and open source servers.
+-   Flattens mediation layers, reducing ad ops burden for managing mediation partners.
+-   Latency is reduced compared to mediation.
 
-This means you **must** have a Prebid Server account in order to use Prebid Mobile. To set up your Prebid Server account for Prebid Mobile, refer to [Get Started with Prebid Server]({{site.github.url}}/prebid-mobile/prebid-mobile-pbs.html)
+## How It Works
 
- 1. As the Prebid Mobile module runs, it fetches bids from your demand partners integrated with Prebid Server.
- 2. Our Prebid Mobile module sends this bid information to your primary ad server SDK using key/value targeting.
- 3. The primary ad server SDK sends a request to the primary ad server including these custom parameters.  These parameters will match the key/value targeting previously configured on Prebid line items by your ad ops teams.
- 4. If the Prebid Mobile bid wins, the primary ad server returns our creative JS to your primary ad server SDK's ad view.
- 5. The creative JS in the ad view fetches the actual cached creative content from Prebid Server. When this happens, the impression is counted.
+Prebid Mobile works in conjunction with Prebid Server, demand partners, and your primary ad server to deliver programmable demand to your mobile app.
+
+### What You Need to Get Started
+
+Before you begin to use Prebid Mobile in your apps, you need to prepare your end-to-end system.
+
+**Mobile Developers**
+
+The following is a one-time setup process for *mobile developers*:
+
+-   [Set up a Prebid Server account]({{site.github.url}}/prebid-mobile/prebid-mobile-pbs.html).
+
+-   Download the Prebid Mobile SDK:
+    -   [SDK for iOS](https://github.com/prebid/prebid-mobile-ios)
+    -   [SDK for Android](https://github.com/prebid/prebid-mobile-android)
+
+After the initial setup, ad slots and line items will need to be set up and maintained:
+
+-   *Mobile developers* register ad units with the Prebid Mobile framework.
+    -   [iOS Code Integration]({{site.github.url}}/prebid-mobile/code-integration-ios.html)
+    -   [Android Code Integration]({{site.github.url}}/prebid-mobile/code-integration-android.html)
+
+**Ad Ops**
```

Comment:
Should Ad Ops go before Mobile Developers?

Reply:
Much of this can happen in any order.  The only hard dependency is: Prebid Server account and configs must be created before the developer can register ad units or request bids through the Prebid Mobile SDK (since those steps require the PBS account ID and PBS config IDs).

Reply:
Much of this can happen in any order. The only hard dependency is: Prebid Server account and configs must be created before the developer can register ad units or request bids through the Prebid Mobile SDK (since those steps require the PBS account ID and PBS config IDs).